### PR TITLE
Fix `torch.nn.functional.normalize` typing on `dim`

### PR DESCRIPTION
**Pull request by puct9**
Docstring and functionality states `dim (int or tuple of ints): the dimension to reduce. Default: 1` but type signature still had `dim: int = 1`, which would be wrong.

### CODE REVIEW

**Comment by puct9** [comment]
Compiler refuses `tuple[int, ...]` part of union type with `Expression of type dots cannot be used in a type expression` hence copying what `torch.nn.functional.pad` did and type only as `list[int]`
